components/base/sensorcontrolled/velocities.go

```diff
@@ -215,6 +226,33 @@ func (sb *sensorBase) State(ctx context.Context) ([]float64, error) {
 	return []float64{linvel.Y, angvel.Z}, nil
 }
 
+func (sb *sensorBase) validateControlLoopConfig(ctx context.Context, controlLoopConfig control.Config) error {
+	sb.blockNames = make(map[string]string)
+	hasLinConst, hasAngConst := false, false
+
+	// Verify linear and angular constant blocks exist and store their names.
+	// These two blocks are the only block names that are used by sensorBase
+	constBlocks, err := sb.loop.ConfigAtType(ctx, "constant")
+	sb.logger.CDebugf(ctx, "const blocks = %v", constBlocks)
```

Comment:
yeah I also don't see a huge difference but all the debugs in this file were recently changed in someone else's PR to use CDebug so I was going for consistency. I will look into Debugw that seems like it might be ideal for this situation

Reply:
looking into it and Debugw doesn't seem necessary here since I'm just logging this one value, but thanks for bringing it to my attention, good think to remember for future cases